docs/blog/arcade/vscode-extension.md

```diff
@@ -0,0 +1,32 @@
+# MakeCode Arcade Extension for Visual Studio Code
+
+**Posted on March 24th, 2023 by [Jaqster](https://github.com/jaqster)**
+
+What?! You heard that right. What has been a back-burner/hackathon/intern project for the last 4 years is actually shipping! [Visual Studio Code](https://code.visualstudio.com) is one of the most popular code editors for professional developers supporting a range of languages and platforms via the [Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace). Now, for more experienced MakeCoders, you can take your MakeCode Arcade games to the next level with Visual Studio Code!
+
+![Extension gallery item for MakeCode Arcade](/static/blog/arcade/vscode-extension/extension-header.png)
+
+You can now import your games into the Visual Studio Code environment, or even start a MakeCode Arcade game from scratch in VS Code. Why would you want to do this? You might if you:
+
+* Are comfortable writing your MakeCode Arcade games in Typescript/JavaScript.
+* Would like access to full language support and intellisense.
+* Have a complex game that you might want to split up into multiple code files.
+* Want to build skills using a professional development tool.
+* Are itching for richer editing tools like syntax highlighting, bracket-matching, auto-indentation, keyboard shortcut mappings and refactoring tools.
+* Want to use [Live Share](https://visualstudio.microsoft.com/services/live-share) or other VS Code extensions.
+
+Here's a look at a game project opened in the MakeCode Arcade Extension:
+
+![MakeCode Arcade extension at work](/static/blog/arcade/vscode-extension/extension.png)
+
+Check out the brief video tour:
+
+https://youtu.be/R-s-ro8qTUs
+
+We are shipping this as a "Preview" because we know there are some improvements we want to make, including an online version with [vscode.dev](https://insiders.vscode.dev/makecode), support for non-named assets, debugging and a bunch of other fixes. To get started, see this brief [setup page](https://arcade.makecode.com/vscode).
```

Comment:
Do you want to expose the actual vscode.dev link at this point?

Reply:
No, let's not point to the vscode.dev route just yet.  It's not quite ready...

Reply:
I just realized that the video I uploaded isn't cropped correctly (black space on the left).  I just uploaded a new video - can you point to this one?  https://youtu.be/EpHsJfztiPU